.github/workflows/ci_cd.yml

```diff
@@ -192,6 +192,12 @@ jobs:
               docker rm $dockerContainers
           }
 
+      - name: "Delete the Docker images (and untagged ones)"
+        if: always()
+        run: |
+          docker image rm ghcr.io/ansys/ansys-sound:latest
```

Comment:
Are you sure you always want to delete the image? I wouldn't do that. Just delete untagged images.
```suggestion
```